sdk/formrecognizer/azure-ai-formrecognizer/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,98 @@
+    )
+    result = poller.result()
+    # process request result
+except HttpResponseError as error:
+    # handle the error here
+```
+
+### Build model error
+Build model errors usually occur when trying to build a custom model. The most common scenarios when this might occur is, if you are building the model with an 
+[Invalid data set](#invalid-training-data-set) or an [Invalid SAS Url](#invalid-sas-url).
+
+#### Invalid training data set
+This error indicates that the provided data set does not match the training data requirements.
+Learn more about building a training data set, [here](https://aka.ms/customModelV3).
+
+Example error output:
+```text
+azure.core.exceptions.HttpResponseError: (InvalidRequest) Invalid request.
+Code: InvalidRequest
+Message: Invalid request.
+Exception Details:      (ModelBuildError) Could not build the model: Can't find any OCR files for training.
+        Code: ModelBuildError
+        Message: Could not build the model: Can't find any OCR files for training.
+```
+
+#### Invalid SAS URL
+This error points to missing permissions on the blob storage SAS URL for the Form Recognizer service to access the training dataset resource.
+
+Example error output:
+```text
+azure.core.exceptions.HttpResponseError: (InvalidArgument) Invalid argument.
+Code: InvalidArgument
+Message: Invalid argument.
+Inner error: {
+    "code": "InvalidSasToken",
+    "message": "The shared access signature (SAS) is invalid: SAS 'list' authorization is missing. Permissions: racwd"    
+}
+```
+
+### Generic Error
+Seeing a "Generic error" returned from the SDK is most often caused by heavy load on the service and throttling of the service. Retrying after backoff time should help mitigate this issue.
+
+Example error output:
+```text
+azure.core.exceptions.HttpResponseError: (3014) Generic error during training.
+Invalid model created with ID=<model ID>
+```
+
+### Unexpected time to build a custom model
+It is common to have a longer completion time than what is expected when building a custom model using the `neural` build mode with `begin_build_document_model()`. Depending on the service load you can usually expect it to take ~10min.
+
+For simpler use-cases, you can use `template` build mode which uses a different model building algorithm that takes less time (typically a few seconds to build). See more about `template` custom models [here](https://aka.ms/custom-template-models). To see more information about `neural` custom models (these models use deep learning to train and build), see documentation [here](https://aka.ms/custom-neural-models).
+
+### Enable HTTP request/response logging
+Reviewing the HTTP request sent or response received over the wire to/from the Azure Form Recognizer service can be useful when troubleshooting issues. To enable logging the HTTP request and response payload, see the example below:
+
+```python
+import logging
+
+# ...
+
+# Acquire the logger
+logger = logging.getLogger('azure.ai.formrecognizer')
```

Comment:
Not sure that we have a named logger in FR, should probably just be "azure". I think I'd either put a full example here of how to enable logging in the client library or just leave the link to the SDK docs.